examples/fitness/lib/user_data.dart

```diff
@@ -0,0 +1,44 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert';
+import 'dart:io';
+import 'dart:async';
+
+import 'package:path/path.dart' as path;
+
+import 'main.dart';
+import 'package:sky/mojo/activity.dart';
+
+String cachedDataFilePath = null;
```

Comment:
I believe that would get rid of my async fanciness.  The first access to the final var would cause a synchronous wait on the mojo ipc.  Unless there is a fancier way to go here?  @sethladd